lib/simplifyExpression/distributeSearch/index.js

```diff
@@ -18,14 +18,50 @@ function distribute(node) {
   if (Node.Type.isUnaryMinus(node)) {
     return distributeUnaryMinus(node);
   }
-  else if (Node.Type.isOperator(node)) {
+  else if (Node.Type.isOperator(node, '*')) {
     return distributeAndSimplifyOperationNode(node);
   }
+  else if (Node.Type.isOperator(node, '^')) {
+    return expandBase(node);
+  }
   else {
     return Node.Status.noChange(node);
   }
 }
 
+// Expand a power node with a non-constant base
+// e.g. (nthRoot(x, 2))^2 -> nthRoot(x, 2) * nthRoot(x, 2)
+// e.g. (2x + 3)^2 -> (2x + 3) (2x + 3)
+function expandBase (node) {
+  // Must be a power node and the exponent must be a constant
+  // Base must either be an nthRoot or sum of terms
+  if (!Node.Type.isOperator(node, '^')) {
+    return Node.Status.noChange(node);
+  }
+
+  const base = Node.Type.isParenthesis(node.args[0])
+        ? node.args[0].content
+        : node.args[0];
+
+  const exponent = node.args[1];
+
+  if (!Node.Type.isConstant(exponent)) {
+    return Node.Status.noChange(node);
+  }
+
+  if (!Node.Type.isFunction(base, 'nthRoot') && !Node.Type.isOperator(base, '+')) {
+    return Node.Status.noChange(node);
+  }
+
+  // If the base is an nthRoot node, it doesn't need the parenthesis
```

Comment:
Definitely not a big deal, so just mentioning this to make you an even more amazing coder: to eliminate code repetition, you could instead do 1) `expandedBase = base if nthRoot else node.args[0]` 2) `expandedNode = Node.creator.operator ... fill(expandedBase)`